doc/user_guide/eds.rst

```diff
@@ -512,11 +512,19 @@ The background can be subtracted from the X-ray intensities with the :py:meth:`~
 Quantification
 --------------
 
-.. versionadded:: 0.8
+Hyperspy now includes three methods for EDS quantification; Cliff-Lorimer, zeta-factors and ionization cross sections.
+
+Quantification must be applied to the background subtracted intensities, which can be found using :py:meth:`~._signals.eds.EDSSpectrum.get_lines_intensity`. The quantification of these intensities can then be determined using the :py:meth:`~._signals.eds_tem.EDSTEMSpectrum.quantification` method. These instensities are a stack of images for each element which can be extracted using :py:meth:`~._signals.eds.EDSSpectrum.get_lines_intensity`. The quantification method, needs be specified as either 'CL', 'zeta', or 'cross_section'. If no method is specified the function will raise an exception.
+A list of factors or cross sections should be supplied in the same order of the listed intesities (please not Hyperspy made using :py:meth:`~._signals.eds.EDSSpectrum.get_lines_intensity` will be in alphabetcial order. The required k-factors can be usually found in the EDS manufacturer software. Where as, zeta-factors and cross sections will need to be determined experimentally using standards. The zeta-factor method is described further in the following papers:
+
```

Comment:
I agree with both points. I think referencing papers for each method is beneficial because then people can look up the exact maths that the function is doing. It stops it being too black box. It also means when they measure their own standards they have a method to follow. I'll see if I can find papers which cover standards for k-factors but I can't think of any off the top of my head.